src/middleware/request_logger.ts

```diff
@@ -1,13 +1,20 @@
 import * as express from 'express';
 // tslint:disable-next-line:no-implicit-dependencies
 import * as core from 'express-serve-static-core';
+import * as HttpStatus from 'http-status-codes';
 
 import { logger } from '../logger';
 /**
  * log middleware
  */
 export function requestLogger(): core.RequestHandler {
     const handler = (req: express.Request, res: express.Response, next: core.NextFunction) => {
+        const origSend = res.send;
+        let cachedBody: any;
+        res.send = (body?: any): express.Response => {
+            cachedBody = body;
+            return origSend.bind(res)(body);
+        };
```

Comment:
Is this necessary? Should we be resetting `res.send` afterwards?

Reply:
Yeah since we need to hook into it, then call the original.